chartfx-chart/src/main/resources/de/gsi/chart/solid-pick.css

```diff
@@ -119,269 +119,201 @@ is being used to size a border should also be in pixels.
+  /* Default buttons color, this is similar to accent but more subtle */
+  -fx-default-button: #ABD8ED;
 
-    /* A bright blue for the focus indicator of objects. Typically used as the
+  /* A bright blue for the focus indicator of objects. Typically used as the
      * first color in -fx-background-color for the "focused" pseudo-class. Also
      * typically used with insets of -1.4 to provide a glowing effect.
      */
-    -fx-focus-color: #039ED3;
-    -fx-faint-focus-color: #039ED322;
+  -fx-focus-color: #039ED3;
+  -fx-faint-focus-color: #039ED322;
 
-    /* The color that is used in styling controls. The default value is based
+  /* The color that is used in styling controls. The default value is based
      * on -fx-base, but is changed by pseudoclasses to change the base color.
      * For example, the "hover" pseudoclass will typically set -fx-color to
      * -fx-hover-base (see below) and the "armed" pseudoclass will typically
      * set -fx-color to -fx-pressed-base.
      */
-    -fx-color: -fx-base;
-
-    /* Chart Color Palette */
-    CHART_COLOR_1: #f3622d;
-    CHART_COLOR_2: #fba71b;
-    CHART_COLOR_3: #57b757;
-    CHART_COLOR_4: #41a9c9;
-    CHART_COLOR_5: #4258c9;
-    CHART_COLOR_6: #9a42c8;
-    CHART_COLOR_7: #c84164;
-    CHART_COLOR_8: #888888;
-    /* Chart Color Palette Semi-Transparent
+  -fx-color: -fx-base;
+  /* Chart Color Palette */
+  CHART_COLOR_1: #f3622d;
+  CHART_COLOR_2: #fba71b;
+  CHART_COLOR_3: #57b757;
+  CHART_COLOR_4: #41a9c9;
+  CHART_COLOR_5: #4258c9;
+  CHART_COLOR_6: #9a42c8;
+  CHART_COLOR_7: #c84164;
+  CHART_COLOR_8: #888888;
+
+  /* Chart Color Palette Semi-Transparent
      * These are used by charts that need semi transparent versions of the above colors, such as BubbleChart. They
      * are exactly the same colors as above just with alpha
      *
      * 20% opacity
      */
-    CHART_COLOR_1_TRANS_20: #f3622d33;
-    CHART_COLOR_2_TRANS_20: #fba71b33;
-    CHART_COLOR_3_TRANS_20: #57b75733;
-    CHART_COLOR_4_TRANS_20: #41a9c933;
-    CHART_COLOR_5_TRANS_20: #4258c933;
-    CHART_COLOR_6_TRANS_20: #9a42c833;
-    CHART_COLOR_7_TRANS_20: #c8416433;
-    CHART_COLOR_8_TRANS_20: #88888833;
-    /* 70% opacity */
-    CHART_COLOR_1_TRANS_70: #f3622db3;
-    CHART_COLOR_2_TRANS_70: #fba71bb3;
-    CHART_COLOR_3_TRANS_70: #57b757b3;
-    CHART_COLOR_4_TRANS_70: #41a9c9b3;
-    CHART_COLOR_5_TRANS_70: #4258c9b3;
-    CHART_COLOR_6_TRANS_70: #9a42c8b3;
-    CHART_COLOR_7_TRANS_70: #c84164b3;
-    CHART_COLOR_8_TRANS_70: #888888b3;
-
-    /***************************************************************************
+  CHART_COLOR_1_TRANS_20: #f3622d33;
+  CHART_COLOR_2_TRANS_20: #fba71b33;
+  CHART_COLOR_3_TRANS_20: #57b75733;
+  CHART_COLOR_4_TRANS_20: #41a9c933;
+  CHART_COLOR_5_TRANS_20: #4258c933;
+  CHART_COLOR_6_TRANS_20: #9a42c833;
+  CHART_COLOR_7_TRANS_20: #c8416433;
+  CHART_COLOR_8_TRANS_20: #88888833;
+
+  /* 70% opacity */
+  CHART_COLOR_1_TRANS_70: #f3622db3;
```

Comment:
Codacy found an issue: [Expected "CHART_COLOR_1_TRANS_70" to be "chart_color_1_trans_70" (property-case)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)